requirements.txt

```diff
@@ -1,15 +1,12 @@
 psutil
 dutil
 prophet==1.0.1
-numpy>=1.15
-pandas>=1.1.5
-xgboost>=1.4.2
-hipe4ml>=0.0.12
+hipe4ml>=0.0.14
 torch>=1.6.0
 scikit-learn>=0.21.3
-onnxconverter-common>=1.6.0
-skl2onnx>=1.7.0
-onnxmltools==1.7.0
-onnxruntime==1.7.0
+onnxconverter-common>=1.9.0
```

Comment:
```suggestion
onnxconverter-common>=1.6.0,<=1.11.0
```

Reply:
See https://github.com/microsoft/hummingbird/issues/613

Reply:
Actually, we don't mind the dependency from onnxruntime I think

Reply:
I missed it indeed, thanks (I wonder why it worked in my local tests..)